dev/devicelab/pubspec.yaml

```diff
@@ -8,67 +8,66 @@ environment:
 
 dependencies:
   args: 1.4.3
-  file: 5.0.0
-  image: 1.1.32
+  file: 5.0.1
+  image: 1.1.14
```

Comment:
I tried doing this over the weekend too but it ended up downgrading this which causes some issues.

Reply:
Good spot, I didn't realise it had gone down. Maybe I'll try setting that one back manually and see what happens.